sdk/core/src/policies/retry_policies/no_retry.rs

```diff
@@ -0,0 +1,17 @@
+use crate::policies::{RetryPolicy, RetryPolicyInstance};
+
+#[derive(Default, Debug, Clone, PartialEq, Eq)]
+pub struct NoRetryPolicy {}
```

Comment:
nit: we might want to give this a private field so that users can't instantiate it directly. This would allow us to add fields in the future without breaking users. This might not be necessary, but better than being forced to not ever be able to add a field.